lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -406,6 +406,30 @@ protected function transformReportbackItem($data) {
   }
 
   /**
+   * Transform Signup data and prepare for API response.
+   *
+   * @param  object  $data Signup object.
+   *   An object containing properties of Signup data:
+   *   - id: (string) The Signup id.
+   *   - created_at: (string) Date Signup was created.
+   *   - campaign: (string) Campaign Signup belongs to.
+   *   - campaign_run: (string) Campaign run Signup belongs to.
+   * @return array
+   */
+  protected function transformSignup($data) {
+    return [
+      'id' => $data->id,
+      'source' => $data->source,
+      'created_at' => $data->created_at,
+      'campaign' => [
```

Comment:
This transformer method should just be for the signup data, so we shouldn't include the campaign info here. That would be a separate call in the `SignupsTransformer` class similar to how the `ReportbackTransformer` includes the campaign data here: https://github.com/DoSomething/phoenix/blob/62106f609af0c415e97559b19b05e9394a600338/lib/modules/dosomething/dosomething_reportback/includes/ReportbackTransformer.php#L79